recipes-kernel/linux/files/0001-add-use-fns.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/security/apparmor/crypto.c b/security/apparmor/crypto.c
```

Comment:
Can you use the whole patches from the ubuntu kernel git tree? I mean whole, as in produced by `git format-patch` for instance, so that it includes the subject, message and sign-off lines.

Also please include something like:
```
Cherry-picked from: <link to the kernel.ubuntu.com>
# eg.
Cherry-picked from: https://kernel.ubuntu.com/git/ubuntu/ubuntu-focal.git/commit/?id=1a432b8c68588a37adc17e47c14a4251638efb39
```

This will make it easier to track down the origin of the patches for both preparing the compliance assets and future updates.

Reply:
Sorry this took a while. I have now created a new PR as my original branch was deleted. https://github.com/morphis/meta-snappy/pull/36

The patches are now updates to contain the Ubuntu references